src/sparsezoo/utils/authentication.py

```diff
@@ -124,24 +55,63 @@ def get_auth_header(
 
     Currently only 'public' authentication type is supported.
 
-    :param authentication_type: authentication type for generating token
```

Comment:
I removed authentication_type as a parameter since there was only 1 valid type and nowhere uses the parameter. Seem okay?